api_intro.py

```diff
@@ -1,18 +1,25 @@
 from datetime import date
+
 import requests
 
 import config
 
 # Client IDs and Secrets from config.py
-ob_client_id = config.OB_CLIENT_ID
-ob_client_Secret = config.OB_CLIENT_SECRET
-sn_client_id = config.SN_CLIENT_ID
-sn_client_Secret = config.SN_CLIENT_SECRET
+try:
+    OB_CLIENT_ID = config.OB_CLIENT_ID
+    OB_CLIENT_SECRET = config.OB_CLIENT_SECRET
+    SN_CLIENT_ID = config.SN_CLIENT_ID
+    SN_CLIENT_SECRET = config.SN_CLIENT_SECRET
+    BASE_URL = config.BASE_URL
+except AttributeError:
+    raise Exception('Config file is missing configuration.')
+
+
+# API urls
+svcnow_url = BASE_URL + 'v1/servicenow/colleges'
+onbase_url = BASE_URL + 'v2/onbase-docs/documents'
+auth_url = BASE_URL + 'oauth2/token'
```

Comment:
Way much cleaner, nice! Just my two cents, ll recommend use f-string to format strings as long as you can, it helps the readibilty a lot.

Reply:
I added to f-strings to these as well.